package.json

```diff
@@ -149,13 +149,13 @@
     "@storybook/theming": "^6.2.8",
     "@visual-snapshot/jest": "^2.0.2",
     "@webpack-cli/serve": "^1.3.0",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.1",
```

Comment:
Can we hard pin this at `0.6.1` since it's a rando fork?

Reply:
Yeah

Reply:
done